consensus/reactor.go

```diff
@@ -796,7 +796,11 @@ OUTER_LOOP:
 			if veEnabled {
 				ec = conR.conS.blockStore.LoadBlockExtendedCommit(prs.Height)
```

Comment:
Note that `LoadBlockExtendedCommit` might panic if the key is not found and the database backend returns error in this case. Opened an issue in CometBFT-db  to fix this behavour: https://github.com/cometbft/cometbft-db/issues/81